tool/profile_visualizer/server.py

```
@@ -0,0 +1,201 @@
+                base_path = self.search_directory
+            dirs = self._list_directories(base_path)
+            self._serve_json(dirs)
+        elif self.path.startswith("/load_file?"):
+            query = urllib.parse.parse_qs(urllib.parse.urlparse(self.path).query)
+            filepath = query.get("path", [""])[0]
+            if filepath and os.path.exists(filepath):
+                try:
+                    with open(filepath, "r", encoding="utf-8") as f:
+                        data = json.load(f)
+                    self._serve_json(data)
+                except Exception as e:  # pylint: disable=broad-except
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)